File: Java/src/main/java/org/usfirst/frc/team3966/robot/commands/TankDrive.java
package org.usfirst.frc.team3966.robot.commands;

import edu.wpi.first.wpilibj.command.Subsystem;
import org.usfirst.frc.team3966.robot.Robot;
import org.usfirst.frc.team3966.robot.values.PS4Buttons;

/**
 *
 */
public class TankDrive extends BaseCommand {

    public TankDrive() {
        super(Robot.drive);
    }

    protected void execute() {
    	System.out.printf("ERROR: This robot only does mecanum drives\n");
        //Robot.drive.tank(Robot.controller.getAxis(PS4Buttons.STICK_LEFT_Y_AXIS), Robot.controller.getAxis(PS4Buttons.STICK_RIGHT_Y_AXIS));

    }
}

File: Python/src/commands/tankdrive.py
"""

Tank drive, using controllers

"""


from wpilib.command import Command

import subsystems

from values import ps4buttons

class TankDrive(Command):

	def __init__(self):
		super().__init__('TankDrive')

		self.requires(subsystems.drive)


	def execute(self):

		_axis = subsystems.oi.joystick.getAxis

		subsystems.drive.tank(_axis(ps4buttons.STICK_LEFT_Y_AXIS), _axis(ps4buttons.STICK_RIGHT_Y_AXIS))




